src/app/instance.h

```diff
@@ -76,23 +76,17 @@ class Instance {
   void InitClientContext();
 
   /// @brief Sets up (or reloads) client/server tunnels
+  /// @param is_reloading Are tunnels configuration in the process of reloading?
   /// @warning Configuration files must be parsed prior to setup
-  void SetupTunnels();
+  void SetupTunnels(bool is_reloading = false);
```

Comment:
I removed this commit. I guess this falls in "// TODO(unassigned): locking etc."